app/lib/upvs_submissions/forms/fuzs_data.rb

```diff
@@ -118,7 +118,7 @@ def join_numbers(building_number, reg_number)
         def load_municipality_identifier
           return unless slovak?
 
-          municipality_code_list_object = CodeList::Municipality.where(value: @municipality&.strip).take
+          municipality_code_list_object = CodeList::Municipality.where("unaccent(value) = unaccent(?)", "#{@municipality.strip}").take
```

Comment:
Na toto treba function index aby to bolo rychlejsie. https://www.postgresql.org/docs/7.3/indexes-functional.html